cmake/projects/cvmatio/hunter.cmake

```diff
@@ -0,0 +1,40 @@
+# This is a header-like file, so include guards needed
+if(DEFINED HUNTER_CMAKE_PROJECTS_CVMATIO_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_CVMATIO_HUNTER_CMAKE_ 1)
+endif()
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_add_version_start)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_add_package)
+
+hunter_add_package(OpenCV)
```

Comment:
This is probably too niche to bother with (especially given opencv complications).  I didn't see any easy way to exclude that one.